jwql/instrument_monitors/common_monitors/bias_monitor.py

```diff
@@ -347,7 +345,7 @@ def most_recent_search(self):
         """
 
         query = session.query(self.query_table).filter(and_(self.query_table.aperture == self.aperture,
-                self.query_table.run_monitor == True)).order_by(self.query_table.end_time_mjd).all()  # noqa: E348 (comparison to true)
+                self.query_table.run_monitor is True)).order_by(self.query_table.end_time_mjd).all()  # noqa: E348 (comparison to true)
 
```

Comment:
Have you checked that this change (to 'is True') works for the query filter logic?

Reply:
This is a bad change. I will revert back to `==` and add a # noqa in line comment. Thanks for catching this one.